packages/diagrams/frontend/sirius-components-diagrams-reactflow/src/renderer/palette/Palette.tsx

```diff
@@ -322,6 +331,15 @@ export const Palette = ({ diagramElementId, onDirectEditClick, isDiagramElementP
       {palette?.toolSections.map((toolSection) => (
         <ToolSection toolSection={toolSection} onToolClick={handleToolClick} key={toolSection.id} />
       ))}
+      {diagramPaletteToolComponents.map((component, index) => {
+        const props: DiagramPaletteToolContributionComponentProps = {
+          editingContextId,
+          diagramId,
```

Comment:
Both could be retrieved from a context already and they are as visible line 170

Reply:
Does the _diagramId_ must be removed from the _canHandle_ too ?
It could be retrieve with the useContext hook.

Reply:
Done